src/bin/mdbook.rs

```diff
@@ -253,19 +253,19 @@ fn serve(args: &ArgMatches) -> Result<(), Box<Error>> {
                 socket.close();
             }}
         </script>
-    "#, public_address, ws_port, RELOAD_COMMAND).to_owned());
+    "#,
+                                public_address,
+                                ws_port,
+                                RELOAD_COMMAND)
+        .to_owned());
```

Comment:
These 5 lines are a little strange, I'm going to check rustfmt to see if there is an option to fix this or if I should file this as a bug.